### ISSUE TITLE
Option to turn off freedraw "smoothing" for better writing experience with pen

### ISSUE THREAD

**Issue by LukaHietala** [excalidraw/excalidraw#6264]
There should be a way to enable "pen mode" when on freedraw mode. And once the pen mode is enable following this should:

- Disable "smoothing processing" when in pen mode
- Palm detection
  - This should probably be implemented by not drawing if one point is already touching the screen, this should also fix the issue with mobile devices 
  - #5176

https://user-images.githubusercontent.com/95122845/219962476-b6822dee-b0d0-4d36-ac29-46dab5b22115.mp4

Related issues:
- https://github.com/excalidraw/excalidraw/issues/6254
- https://github.com/excalidraw/excalidraw/issues/4802
- https://github.com/excalidraw/excalidraw/issues/6088

**Comment by ctwhome** [comment]
thank you for this issue, this is exactly what I wanted to say with #6254

**Comment by LukaHietala** [comment]
@dwelle Any thoughts?

**Comment by ctwhome** [comment]
This limits handwritten note-taking in tablets (Galaxy tabs, Surface, iPads), it looks like it is only a matter of a few adjustments when detecting the pen, right?

**Comment by nonoMain** [comment]
hey! any updates on this?
I really want to be able to use excalidraw for my notes (right now I use it only for diagrams and such)

**Comment by ash-paragram** [comment]
@nonoMain yeah, I want to be able to use excalidraw for all my note-taking with stylus, just because my obsidian organisation system is so robust having a note-taking utility integrated would be a godsend

**Comment by zhraa11** [comment]
It's so uncomfortable that I can't put my hands on the tablet.

**Comment by EDLLT** [comment]
Any updates on this?
